README.adoc

```diff
@@ -107,3 +107,7 @@ You can configure the policy with the following options:
 | Outgoing message cannot be transformed properly to XML
 
 |===
+
+=== Nested objects
+
+To limit the processing time in case of nested object, a default max depth of nested object has been defined to 1000. This default value can be overriden using the environment variable `gravitee_policy_json_xml_maxdepth`.
```

Comment:
In XML-To-json policy we set it to 100, maybe we should keep 100 to be consistent.
by the way, a depth of 100 is already big, so it should be sufficient as a default value ?
wdyt ?

Reply:
I've cherry picked the commit from `master` 🤔 
I can make the change but I will need to do it on `master` and `1.x` branches 🤷